src/applications/representative-appoint/tests/containers/ConfirmationDigitalSubmission.unit.spec.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { expect } from 'chai';
+import { render, fireEvent } from '@testing-library/react';
+import sinon from 'sinon';
+import { $ } from 'platform/forms-system/src/js/utilities/ui';
+import moment from 'moment';
+import ConfirmationDigitalSubmission from '../../containers/ConfirmationDigitalSubmission';
+import mockFormData from '../fixtures/data/form-data.json';
+
+const mockStore = state => ({
+  getState: () => state,
+  subscribe: () => {},
+  dispatch: () => ({}),
+});
+
+describe('<ConfirmationDigitalSubmission>', () => {
+  const getProps = () => {
+    return {
+      props: {
+        formData: {
+          submission: {
+            timestamp: moment()
+              .subtract(5, 'days')
+              .toISOString(),
+          },
+          data: mockFormData,
+        },
+      },
+      mockFormStore: mockStore({
+        form: {
+          submission: {
+            timestamp: moment()
+              .subtract(5, 'days')
+              .toISOString(),
+          },
+          data: mockFormData,
+        },
+      }),
+    };
+  };
+
+  const renderContainer = (props, mockFormStore) => {
+    return render(
+      <Provider store={mockFormStore}>
+        <ConfirmationDigitalSubmission {...props} />
+      </Provider>,
+    );
+  };
+
+  it('should render the component', () => {
+    const { props, mockFormStore } = getProps();
+    const { container } = renderContainer(props, mockFormStore);
```

Comment:
Can this go into a `beforeEach` block because it's being repeated in each example?

```
    const { props, mockFormStore } = getProps();
    const { container } = renderContainer(props, mockFormStore);
```